libr/core/cmd_search.c

```diff
@@ -11,6 +11,12 @@ static int cmd_search(void *data, const char *input);
 
 #define USE_EMULATION 0
 
+// Size needed to search a AES key.
+#define AES_SEARCH_LENGTH 40
+
+// Size meeded to search a private key.
```

Comment:
meeded is not english :P anyway i think the variable namne is meaningful enough, no need for the comments

Reply:
yeah I agree, the names should be already descriptive enough in this case.